### ISSUE TITLE
why I can't do myJson[myJson.count]=someValue ?

### ISSUE THREAD

**Issue by djdance** [SwiftyJSON/SwiftyJSON#782]
sorry if I'm just asking a newbee question but I still for a year can't properly understand why I can't do

```
some loop {
  myJson[myJson.count]=someValue
}
print("count=\(myJson.count)") //here always 0
```

I have to use myJson["\\(myJson.count)"], but the Readme says I can use [int]. What's wrong?

**Comment by TungDiamond** [comment]
hi sor ... my blockchain can't recieve any payments form any one .. Please
help me to fix it .. i need it very much ...

2017-01-25 3:47 GMT+07:00 denis <notifications@github.com>:

**Comment by TungDiamond** [comment]
I'm very sorry . but i hope you could help me for fixing it  in vietnam , i have very less information . so i'm apologize but please , my btc are pended on API board but i don't know how to get it back .
please .

**Comment by wongzigii** [comment]
Hey @djdance Sorry for the late response. Could you provide a specific sample to reproduce it?

**Comment by wongzigii** [comment]
I assume this issue can be closed since #793 would fix the mutability. Thanks for reporting this.
